doc/nrf/libraries/caf/sensor_manager.rst

```diff
@@ -160,6 +165,52 @@ To use the sensor trigger, complete the following steps:
 .. note::
     |only_configured_module_note|
 
+Enabling passive power management
+=================================
+
+The |sensor_manager| can react to ``power_down_event`` and ``wake_up_event``.
+This functionality is called passive power management.
+
+To use the passive power management in |sensor_manager|, complete the following steps:
+
+1. Enable :kconfig:`CONFIG_CAF_SENSOR_MANAGER_PM` option.
+   If a sensor of your choice supports trigger functionality,its enabled and configured by you, there is nothing more to do.
+   If you did not enable trigger functionality for your sensor, continue with next steps.
+#. Extend the module configuration file by adding :c:member:`sensor_config.suspend_pm_state` in an array of :c:struct:`sensor_config`.
+   :c:member:`sensor_config.suspend_pm_state` configures sensor's device power state in which it would be put when ``power_down_event`` is received.
+   The default value for :c:member:`sensor_config.suspend_pm_state` is :c:enumerator:`PM_DEVICE_STATE_ACTIVE`.
+   For more details about device power states in Zephyr refer to :ref:`zephyr:power_management_api`.
+
+   For example, the extended configuration file for the LIS2DH accelerometer could look like follows:
+
+   .. code-block:: c
+
+        #include <caf/sensor_manager.h>
+
+        static const struct sensor_config sensor_configs[] = {
+                {
+                        .dev_name = "LIS2DH12-ACCEL",
+                        .event_descr = "accel_xyz",
+                        .chans = accel_chan,
+                        .chan_cnt = ARRAY_SIZE(accel_chan),
+                        .sampling_period_ms = 20,
+                        .active_events_limit = 3,
+                        .trigger = &trig,
```

Comment:
It should not be here. Select one: trigger or suspend_pm_state.

Reply:
Removed.